Assets/Scripts/Player/PlayerMovement.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[RequireComponent(typeof(CharacterController))]
+public class PlayerMovement : MonoBehaviour
+{
+    [SerializeField] private float m_moveSpeed = 5f;
```

Comment:
Same thing as everywhere else, remove in-code initializers and extract direct code from monoBehaviour methods to their own.